pyfar/coordinates.py

```diff
@@ -1856,26 +1847,27 @@ def cyl2cart(azimuth, height, radius):
         0 < azimuth < 2 \\pi
 
 
-    Notes
-    -----
-    To ensure proper handling of the azimuth angle, the arctan2 implementation
-    from numpy is used.
+    .. note::
+        To ensure proper handling of the azimuth angle, the arctan2
+        implementation from numpy is used.
```

Comment:
Notes instead? Not consistent with `cart2cyl`

Reply:
Yes - did that.